### ISSUE TITLE
Button control does not support UIA accessibility

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3214]
* .NET Core Version: 

.NET Core 5.0
Version: 5.0.100-alpha.1.20073.10
Commit: 29f4d693a9
Runtime Environment:
OS Name: Windows
OS Version: 10.0.18363
OS Platform: Windows
RID: win10-x64
Base Path: C:\Program Files\dotnet\sdk\5.0.100-alpha1-05536\
Host (useful for support):
Version: 5.0.0-alpha.1.20072.3
Commit: c3dc1fdfdc

* Have you experienced this same bug with .NET Framework?: 

Yes

**Problem description:**

![image](https://user-images.githubusercontent.com/23213980/81074322-84882980-8ef1-11ea-98c1-ec750b5ea3a1.png)

Accessibility providers:
[pid:43924,providerId:0x12AC8 Main:Nested [pid:1456,providerId:0x12AC8 Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)];
Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]"

**Expected behavior:**
There should be UIA accessibility provider from WinForms

**Minimal repro:**
Create WinForms app with Button control and analyze it using Inspect.

**Related item:**
This work item is the sub of #2164

**Comment by Ashley-Li** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, the issue has been fixed that there's UIA accessibility provider from WinForm.Please see below result:
![image](https://user-images.githubusercontent.com/56664234/98623034-b4105800-22bf-11eb-81cf-0fc0aeef90b5.png)
